Riot/Modules/Room/RoomViewController.m

```diff
@@ -200,6 +200,8 @@ @interface RoomViewController ()
 
     // Observe kRiotDesignValuesDidChangeThemeNotification to handle user interface theme change.
     id kRiotDesignValuesDidChangeThemeNotificationObserver;
+    
+    BOOL isInReplyMode;
```

Comment:
It probably worth to describe here what is this mode: If the user has selected a message, we are in this mode.